pkg/cloud-provider/cloudapi/aws/aws_cloudinterface_impl.go

```diff
@@ -105,14 +105,14 @@ func (c *awsCloud) GetAccountStatus(accNamespacedName *types.NamespacedName) (*c
 	return c.cloudCommon.GetStatus(accNamespacedName)
 }
 
-// AddInventoryPoller adds poller for polling cloud inventory.
-func (c *awsCloud) AddInventoryPoller(accountNamespacedName *types.NamespacedName) error {
-	return c.cloudCommon.AddInventoryPoller(accountNamespacedName)
+// DoInventoryPoll calls cloud API to get vm and vpc resources.
+func (c *awsCloud) DoInventoryPoll(accountNamespacedName *types.NamespacedName) error {
+	return c.cloudCommon.DoInventoryPoll(accountNamespacedName)
 }
 
-// DeleteInventoryPoller deletes an existing poller created for polling cloud inventory.
-func (c *awsCloud) DeleteInventoryPoller(accountNamespacedName *types.NamespacedName) error {
-	return c.cloudCommon.DeleteInventoryPoller(accountNamespacedName)
+// DeleteInventoryPoll resets cloud snapshot to nil.
+func (c *awsCloud) DeleteInventoryPoll(accountNamespacedName *types.NamespacedName) error {
```

Comment:
nit: DeleteInventoryPollCache or DeleteInventoryCache

Reply:
For me, DeleteInventoryPollCache seems better.